OpenOversight/app/models.py

```diff
@@ -188,12 +188,25 @@ def job_title(self):
                 self.assignments_lazy, key=lambda x: x.star_date or date.min
             ).job.job_title
 
+    def unit_descrip(self):
+        if self.assignments_lazy:
+            return max(
+                self.assignments_lazy, key=lambda x: x.star_date or date.min
+            ).unit.descrip
+
     def badge_number(self):
         if self.assignments_lazy:
             return max(
                 self.assignments_lazy, key=lambda x: x.star_date or date.min
             ).star_no
 
+    def currently_on_force(self):
+        if self.assignments_lazy:
+            most_recent = max(
+                self.assignments_lazy, key=lambda x: x.star_date or date.min
+            )
+            return "Yes" if most_recent.resign_date is None else "No"
```

Comment:
This is missing the "Uncertain" branch from the original implementation. Not sure if it matters?

```suggestion
        if self.assignments_lazy:
            most_recent = max(
                self.assignments_lazy, key=lambda x: x.star_date or date.min
            )
            return "Yes" if most_recent.resign_date is None else "No"
        return "Uncertain"
```

Oh never mind, I see it's in the template now... 😕 I think I prefer that kind of thing in the property function itself cause it makes it actually unit testable, but I suppose it's better to just follow the already established pattern.

Reply:
Ah, that's a great point! I'll add that